README.md

```diff
@@ -134,6 +134,9 @@ For `labelRenderer`, you can provide a full path - [see this PR](https://github.
 - `shouldExpandNode: function(keyName, data, level)` - determines if node should be expanded (root is expanded by default)
 - `hideRoot: Boolean` - if `true`, the root node is hidden.
 - `sortObjectKeys: Boolean | function(a, b)` - sorts object keys with compare function (optional). Isn't applied to iterable maps like `Immutable.Map`.
+- `onNodeExpansionChanging: function(keyName, data, level, expanded)` - invoked when a node is expanding or collapsing.
+- `onNodeExpansionChanged: function(keyName, data, level, expanded)` - invoked when a node is expanded or collapsed.
+- `isNodeExpansionDynamic: Boolean` - if `true`, `shouldExpandNode` will be called each render to determine the expansion state of a node, rather than just once during mount
```

Comment:
This one won't be necessary with https://github.com/alexkuz/react-json-tree/pull/61 I suppose.